src/core/trulens/core/database/orm.py

```diff
@@ -315,6 +319,68 @@ def parse(
                     multi_result=obj.multi_result,
                 )
 
+        class GroundTruth(base):
```

Comment:
This seems pretty generic, shouldn't it just be "DatabaseRow" or something like that?

Reply:
I agree the naming properly can be improved. 

That said, the reason I specifically use GroundTruth is to denote that this is storing and managing the same `ground_truth` (semantically, with formats being variable) that's being expected as an argument by the GroundTruthAgreement feedback function. does it make sense?